msg/messages.js

```diff
@@ -975,6 +975,23 @@ Blockly.Msg.LISTS_GET_SUBLIST_TAIL = '';
 /// [[File:Blockly-get-sublist.png]]
 Blockly.Msg.LISTS_GET_SUBLIST_TOOLTIP = 'Creates a copy of the specified portion of a list.';
 
+/// url - Infomation describing sorting a list.
+Blockly.Msg.LISTS_SORT_HELPURL = 'https://github.com/google/blockly/wiki/Lists#sorting-a-list';
+/// Sort a list of items.
+Blockly.Msg.LISTS_SORT_TITLE = 'sort %1 %2 %3';
+/// tooltip - See [https://github.com/google/blockly/wiki/Lists#sorting-a-list
+Blockly.Msg.LISTS_SORT_TOOLTIP = 'Sort a list.';
+/// sorting order or direction from low to high values
+Blockly.Msg.LISTS_SORT_ORDER_ASCENDING = 'ascending';
+/// sorting order or direction from high to low values
+Blockly.Msg.LISTS_SORT_ORDER_DESCENDING = 'descending';
+/// sorting by treating each item as a numeric value.
+Blockly.Msg.LISTS_SORT_TYPE_NUMERIC = 'numeric';
+/// sorting by treating each item as text.
```

Comment:
Suggest: "...each item alphabetically."  As text does not mean much to a non CS person.